pom.xml

```
@@ -54,22 +54,24 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Java Version Dependencies -->
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
 
         <!-- Arquillian Dependencies -->
-        <version.arquillian>1.7.0.Alpha14</version.arquillian>
-        <payara.arquillian.container.version>3.0.alpha8</payara.arquillian.container.version>
+        <version.arquillian>1.9.5.Final</version.arquillian>
+        <payara.arquillian.container.version>3.1</payara.arquillian.container.version>
 
         <!-- Embedded Dependencies -->
-        <tyrus.version>2.0.0</tyrus.version>
+        <tyrus.version>2.2.0</tyrus.version>
         <jersey.version>3.0.3.payara-p1</jersey.version>
         <cdi.version>4.0.1</cdi.version>
         <jaxb.version>4.0.0</jaxb.version>
+        <shrinkwrap.version>3.3.4</shrinkwrap.version>
+        <jakarta.json.version>2.1.3</jakarta.json.version>
 
         <!-- Payara Dependencies -->
         <payara.version>6.2023.1</payara.version>
```

If we're making a Payara 7 specific branch we should update this as well
```suggestion
        <payara.version>7.2025.1.Alpha3-SNAPSHOT</payara.version>
```